cli/lib/kontena/cli/stacks/yaml/validations.rb

```diff
@@ -43,7 +43,7 @@ def common_validations
               )
             end
           elsif value.is_a?(Array)
-            value.all? { |v| v.kind_of?(String) && v =~ /\A\S+(?<!\\)=.*/ }
+            value.all? { |v| v.kind_of?(String) && v =~ /^([^=]+)=(.*)/ }
```

Comment:
What's the best regexp to validate a sane env, without rejecting crazy edge cases that people might actually be using?